content/docs/getting-started/cms-set-up.md

```diff
@@ -56,9 +67,17 @@ export default App;
 
 ## Configure the CMS object
 
-When instantiating the CMS, you can pass [options](/docs/cms#cms-configuration) to configure how the CMS object works.
+The `tinacms` package provides two possible user interfaces: the sidebar and the toolbar.
```

Comment:
```suggestion
```

Reply:
This seems off, as there is more to the user interface with inline editing and possibly even more UI components later down the line.